src/devices/sound/rf5c400.cpp

```diff
@@ -230,6 +238,11 @@ void rf5c400_device::sound_stream_update(sound_stream &stream, std::vector<read_
 		env_step = channel->env_step;
 		env_rstep = env_step * channel->env_scale;
 
+		if (start == end) {
```

Comment:
This is a K&R format file (unlike the Allman files you were working on recently).  Could you move the opening brace to a line of its own, or remove the braces since there’s only a single statement in the controlled block?

Reply:
Done. Also found another line where I did this same thing.